hypha/public/utils/models.py

```diff
@@ -264,6 +271,7 @@ class Meta:
         MultiFieldPanel([
             FieldPanel('site_logo_default'),
             FieldPanel('site_logo_mobile'),
```

Comment:
Not related to the PR but these should be [ImageChooserPanel](https://docs.wagtail.io/en/stable/reference/pages/panels.html#module-wagtail.images.edit_handlers) for easily choosing images instead of a dropdown.

Reply:
A lot nicer, thanks!